timesketch/frontend-ng/src/components/LeftPanel/SigmaRule.vue

```diff
@@ -19,36 +19,78 @@ limitations under the License.
       <div @click="expanded = !expanded" style="cursor: pointer; font-size: 0.9em">
         <v-icon v-if="!expanded">mdi-chevron-right</v-icon>
         <v-icon v-else>mdi-chevron-down</v-icon>
-        {{ sigmaRule.title }}
+        {{ sigmaRule.title }}<v-chip rounded x-small class="ml-2">{{ sigmaRule.status }}</v-chip>
       </div>
       <v-spacer></v-spacer>
-      <div><v-icon>mdi-dots-vertical</v-icon></div>
+      <div>
+        <v-menu>
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn small icon v-bind="attrs" v-on="on">
+              <v-icon>mdi-dots-vertical</v-icon>
+            </v-btn>
+          </template>
+          <v-card>
+            <v-list>
+              <v-list-item-group>
+                <v-list-item v-on:click="editSigmaRule(sigmaRule.rule_uuid)">
+                  <v-list-item-icon>
+                    <v-icon>mdi-pencil </v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>Edit Rule</v-list-item-title>
+                  </v-list-item-content>
+                </v-list-item>
+                <v-list-item v-on:click="deprecateSigmaRule(sigmaRule.rule_uuid)">
+                  <v-list-item-icon>
+                    <v-icon>mdi-archive</v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>Do not use the rule in Sigma analyzer</v-list-item-title>
```

Comment:
hm your suggestion is not really indicating that by pressing, this would be done to the rule. But jbn and myself had no good idea either how to name that button that would also indicate what it really does.

Reply:
I'm not 100% on the wording, but I think that Tom's suggestion is clearer, so +1 on the suggestion